From e2e2f5a7b783ee8a27d861369ba9de984b7d7221 Mon Sep 17 00:00:00 2001
From: Tim Foster <tim.foster@joyent.com>
Date: Wed, 6 Feb 2019 19:46:54 +0000
Subject: [PATCH] TOOLS-2174 agent-cache 'clone' should use '--' when switching
 branches Reviewed by: Trent Mick <trentm@gmail.com> Approved by: Trent Mick
 <trentm@gmail.com>

---
 deps/eng | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/deps/eng b/deps/eng
index 4443fcd..7c472bc 160000
--- a/deps/eng
+++ b/deps/eng
@@ -1 +1 @@
-Subproject commit 4443fcd38fe1682556e1851df97c7d652d078f3d
+Subproject commit 7c472bc495ba44ac4cdde0af50ff627021942524
-- 
2.21.0

